packages/headless/src/test/mock-product-listing-state.ts

```diff
@@ -1,3 +1,4 @@
+import {getProductListingInitialState} from '../features/commerce/product-listing/product-listing-state';
 import {getConfigurationInitialState} from '../features/configuration/configuration-state';
```

Comment:
You should be able to remove this file completely; there should not be any more references to its exported `buildMockProductListingState` function once we remove the code of the product listing sub-package.